esmvaltool/recipes/recipe_collins13ipcc.yml

```diff
@@ -0,0 +1,2539 @@
+###############################################################################
```

Comment:
Please use the tool `yamllint` to check that your recipe is correct. However, feel free to ignore any complaints about line too long or too many spaces if you think your formatting is more readable.